kuma/static/styles/components/structure/page-header/main-header.scss

```diff
@@ -31,6 +31,17 @@ Main header, wraps all components of logo, main, and secondary navigation
     }
 }
 
+a[class*='logoContainer'] {
+    outline: none;
```

Comment:
Will this still be necessary when we change the focus to the skip-link container? Do we want to have any sort of visible styling when non `focus-visible` users are tabbing through the page?

Reply:
> Will this still be necessary when we change the focus to the skip-link container? 

Nope

> Do we want to have any sort of visible styling when non focus-visible users are tabbing through the page?

Where it makes sense for sure, but the main focus(excuse the pun ;p) of these are for users using a keyboard. For the most part, knowing where focus is when using a mouse is generally intuitive. There are places where we can improve though. I often find that having an `active` state or, clear indication when a clicked was registered is super useful.